tests/adapters/test_ella_adapter.py

```diff
@@ -0,0 +1,41 @@
+import torch
+
+import refiners.fluxion.layers as fl
+from refiners.fluxion.utils import no_grad
+from refiners.foundationals.latent_diffusion import SD1ELLAAdapter, SD1UNet
+from refiners.foundationals.latent_diffusion.ella_adapter import ELLACrossAttentionAdapter
+
+
+def new_adapter(target: SD1UNet) -> SD1ELLAAdapter:
+    return SD1ELLAAdapter(target=target)
+
+
+@no_grad()
+def test_inject_eject(test_device: torch.device):
+    unet = SD1UNet(in_channels=4, device=test_device, dtype=torch.float16)
+    initial_repr = repr(unet)
+    adapter = new_adapter(unet)
+    assert repr(unet) == initial_repr
+    adapter.inject()
+    assert repr(unet) != initial_repr
+    adapter.eject()
+    assert repr(unet) == initial_repr
+    adapter.inject()
+    assert repr(unet) != initial_repr
+    adapter.eject()
+    assert repr(unet) == initial_repr
+
+
+@no_grad()
+def test_scale(test_device: torch.device):
```

Comment:
```suggestion
def test_ella_cross_attention(test_device: torch.device):
```

Reply:
Since there is no scale involved (it's rather testing/inspecting that the adapter properly modified the cross-attentions)